core/src/main/java/hudson/slaves/NodeProvisioner.java

```diff
@@ -191,23 +191,21 @@ private void update() {
         provisioningLock.lock();
         try {
             lastSuggestedReview = System.currentTimeMillis();
-
-            // We need to get the lock on Queue for two reasons:
```

Comment:
Maybe keep this block comment in original indentation to avoid the need for a `-w` diff.